Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.91.3"
+version = "0.92.0"
```

Comment:
Is this a breaking change?

Reply:
I think so because `BoundaryCondition` is part of the API right (even though it is rarely directly used)?

So we change the syntax `BoundaryCondition(Value, 1)` to `BoundaryCondition(Value(), 1)`, although we might be able to make it not breaking by defining:
```julia
BoundaryCondition(Classification::DataType, args...) = BoundaryCondition(Classification(), args...)
```

?